tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -803,10 +806,11 @@ def startTestServers(
         return test_servers
 
     def startTestClient(
-        self, test_server: XdsTestServer, **kwargs
+        self, test_server: XdsTestServer, server_target=None, **kwargs
```

Comment:
I think eventually we should move all tests from `startTestClient(test_server)` to startTestClient(server_target).

But for now I don't like that the test server is passed from `GammaXdsKubernetesTestCase` for no reason.

Let's make a private method

`def _start_test_client(server_target: str, **kwargs)`

And have both this `startTestClient` and the one in `GammaXdsKubernetesTestCase` call it.